homeassistant/components/homewizard/config_flow.py

```diff
@@ -66,11 +66,14 @@ async def async_step_user(
                     data=user_input,
                 )
 
+        user_input = user_input or {}
         return self.async_show_form(
             step_id="user",
             data_schema=Schema(
                 {
-                    Required(CONF_IP_ADDRESS): str,
+                    Required(
+                        CONF_IP_ADDRESS, default=user_input.get(CONF_IP_ADDRESS)
```

Comment:
None is accepted here right?

Reply:
Not _really_ sure, but PyPi approved it 🤷 

Got this style from here; https://github.com/home-assistant/core/blob/670bd97777537fe51df38a3bae52d7249ed87b63/homeassistant/components/deluge/config_flow.py#L59-L75

Reply:
Maybe its good to add a test case for this?

Reply:
Added test case to validate user input is added, no user input is rejected by the scheme itself.